src/_Concert.scss

```diff
@@ -0,0 +1,75 @@
+
+
+* {
+  box-sizing: border-box;
+}
+
+.concert { 
+  overflow: hidden;
+  height: 200px;
+  width: 250px;
+  color: black;
+  border-radius: 3%;
+/*  background-color: #fff;
+  border: 1px solid black;
```

Comment:
We'll want to clean out commented out lines eventually but we can do a linting/cleaning run through everything as we get towards the end

Reply:
⬆️